components/EventCarousel.tsx

```diff
@@ -0,0 +1,86 @@
+"use client";
+import { IMAGES } from "@/libs/constant/image.constant";
+import { StaticImport } from "next/dist/shared/lib/get-img-props";
+import Image from "next/image";
+import React from "react";
+import Slider, { Settings } from "react-slick";
+
+export default function EventCarousel() {
```

Comment:
Instead of redundant Carousel code and settings , create a CustomCarousel in components/shared and pass the necessary data as params to the carousel
Also include an option to override settings in case along with params